doc/classes/ProjectSettings.xml

```diff
@@ -775,6 +775,33 @@
 		<member name="internationalization/locale/test" type="String" setter="" getter="" default="&quot;&quot;">
 			If non-empty, this locale will be used when running the project from the editor.
 		</member>
+		<member name="internationalization/pseudolocalization/double_vowels" type="bool" setter="" getter="" default="false">
+			Double vowels in strings during pseudolocalization to simulate the lengthening of text due to localization.
+		</member>
+		<member name="internationalization/pseudolocalization/expansion_ratio" type="float" setter="" getter="" default="0.0">
+			Expansion ratio to be used during pseudolocalization. A value of 0.3 is sufficient for most practical purposes.
+		</member>
+		<member name="internationalization/pseudolocalization/fake_bidi" type="bool" setter="" getter="" default="false">
+			Fake RTL text.
+		</member>
+		<member name="internationalization/pseudolocalization/override" type="bool" setter="" getter="" default="false">
+			Replace all characters in the string with '*'. Useful for finding non-localizable strings.
+		</member>
+		<member name="internationalization/pseudolocalization/prefix" type="String" setter="" getter="" default="&quot;[&quot;">
+			Prefix that will be prepended to the pseudolocalized string.
+		</member>
+		<member name="internationalization/pseudolocalization/replace_with_accents" type="bool" setter="" getter="" default="true">
+			Replace all characters with their accented variants during pseudolocalization.
+		</member>
+		<member name="internationalization/pseudolocalization/skip_placeholders" type="bool" setter="" getter="" default="true">
+			Skip placeholders for string formatting like %s, %f, etc during pseudolocalization. Useful to identify strings which need additional control chars to display correctly.
+		</member>
+		<member name="internationalization/pseudolocalization/suffix" type="String" setter="" getter="" default="&quot;]&quot;">
+			Suffix that will be appended to the pseudolocalized string.
+		</member>
+		<member name="internationalization/pseudolocalization/use_pseudolocalization" type="bool" setter="" getter="" default="false">
+			Turn on Pseudolocalization for the project.
```

Comment:
```suggestion
			If [code]true[/code], enables pseudolocalization for the project. This can be used to spot untranslatable strings or layout issues that may occur once the project is localized to languages that have longer strings than the source language.
			[b]Note:[/b] This property is only read when the project starts. To toggle pseudolocalization at run-time, use [method TranslationServer.set_pseudolocalization_enabled] instead.
```